### ISSUE TITLE
common and watch options don't play well together

### ISSUE THREAD

**Issue by eggyal** [atomify/atomify-js#62]
The [documentation for `opts.common`](https://github.com/atomify/atomify-js#optscommon) states:

> If using a callback, you're passed an object that with keys of each entry file and values of the compiled JS file.
> 
> <pre>[ <strong><em>deletia</em></strong> ]</pre>
> 
> You'll need to listen to the 'entry' event to get the compiled entry files.

However, when combined with [`opts.watch`](https://github.com/atomify/atomify-js#optswatch), neither of the above statements are true: the callback receives only a single bundle, and the `'entry'` event is not called.

**Comment by joeybaker** [comment]
Yes, we should do a better job documenting this in the readme. The two options currently are must ally exclusive.
